cime_config/usermods_dirs/NEON/GUAN/shell_commands

```diff
@@ -1,3 +1,4 @@
 ./xmlchange NEONSITE=GUAN
 ./xmlchange PTS_LON=293.13112
 ./xmlchange PTS_LAT=17.96882
+./xmlchange RUN_STARTDATE=2018-06-01
```

Comment:
@wwieder It will just start on June/1st and use that as it's reference date. This means for purposes of spinup dynamics June/1st is the date when spinup variables are updated. It's sort of like making June/1st the start of your fiscal year. Somethings are still tied to the calendar year (like dynamic land use change), but anything that is measuring time since start of run or doing running averages will use June/1st as the marker. Does that help?

Reply:
ok, but for spinup, when you cycle over years 2018-2020 repeatedly does the model get to May 30 2020 of the last year and then restart on June 1 2018 or does it go all the way until Dec 31 2020 and then restart on June 1 2018?